build_process/node.md

```diff
@@ -1,19 +1,59 @@
 # Node
 
-## NPM Scripts
+## Managing Your Local Node Versions
 
-Our projects use different build systems (Gulp, Grunt, custom, etc), but many 
-times they have tasks that do similar things on every project. As developers 
-move from project to project we can save some cognitive overhead by 
-standardizing the commands we use for these common tasks. With node we can use 
-[npm scripts](https://docs.npmjs.com/misc/scripts) as shortcuts to our various 
+To manage your own Node versions we have a few recommendations, but you should do what works for you. Whichever method is reliable and comfortable is the right one.
+
+### Happy Path
+
+Use [`nvm`](https://github.com/nvm-sh/nvm). See documentation for [installation](https://github.com/nvm-sh/nvm#installation-and-update) and [usage](https://github.com/nvm-sh/nvm#usage). 
+
+### Secondary Options
+
+- [`n`](https://www.npmjs.com/package/n) - this is an `npm` install-able node-based tool. It's real nice.
+- [`brew`](https://brew.sh/) - this is a perfectly fine option. See [instructions](./managing-node-with-brew.md) for using `brew`.
+
+## Project Node Versions
+
+All projects that rely on Node for build process or otherwise should use an [LTS release](https://github.com/nodejs/Release). The specific required version should be clearly defined for developers.
+
+The `package.json` for any given project _should_ specify the required `node` version in the `engines` field. This field supports semver ranges. It looks like this...
+```json
+{
+  "engines": {
+    "node": "^10.0.0"
+  }
+}
+```
+
+## Version Automation
+
+There are a few things we can do to help automate and/or enforce version requirements on a per-project basis. 
+
+### Happy Path
+
+Install [`node`](https://www.npmjs.com/package/node) as a `devDependency`. You can add the latest LTS release with `npm i -D node@lts`. With this, all `npm run` scripts (as well as [`npx`](https://www.npmjs.com/package/npx) run from the project root) will use the locally installed binary.
+
+### Secondary Options 
+
+- Use a [`.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc) file. This will work for those who use [`nvm`](https://github.com/nvm-sh/nvm) to manage versions. The `nvm use` command will automatically pick up the version specified in `.nvmrc`.
+- Use the [`check-node-version`](https://github.com/parshap/check-node-version) package to check node, npm, or yarn versions in a `prestart` script. This will not automatically change node versions, but will add a fast failure when running `npm start` so developers know immediately of a need to switch versions. Use the `--package` flag to read version requirements directly from the `engines` field in `package.json`.
```

Comment:
I believe we should be using this on all projects. It ensures we're using the correct runtime when deployed
.

Reply:
It also ensures `package.json` is configured with `engines` as you mention above.

Reply:
Good points. Maybe this should under the primary solutions as opposed to secondary?

Reply:
I would be 💯 on board with it being part of the happy path.

Reply:
Moved to happy path.